client/extensions/woocommerce/app/dashboard/setup-header.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const SetupHeader = ( { imageSource, imageWidth, subtitle, title } ) => {
+	return (
+		<div className="dashboard__setup-header">
+			{ imageSource ? <img src={ imageSource } width={ imageWidth } className="dashboard__setup-header-image" /> : null }
+			{ <h2 className="dashboard__setup-header-title">{ title }</h2> }
+			{ subtitle ? <h3 className="dashboard__setup-header-subtitle">{ subtitle }</h3> : null }
```

Comment:
LOL :) Fixed in https://github.com/Automattic/wp-calypso/pull/14202/commits/66d13147f190def3689b815188fde46da04a36b4